jupyterlab_wipp/jupyterlab_wipp/wipp.py

```diff
@@ -99,7 +100,66 @@ def get_image_collections_all_pages(self):
 
     def get_image_collections(self):
         return [collection.json for collection in sum(self.get_image_collections_all_pages(), [])]
+
+    def search_image_collections_summary(self, name):
+        r = requests.get(os.path.join(self.api_route, f'imagesCollections/search/findByNameContainingIgnoreCase?name={name}'))
```

Comment:
Can we add error handling to all the APi calls?

Reply:
done